### PR TITLE
DOCS: review pr on local system

### LINKED ISSUE #223: [DOCS] How to review the PR by running it on our local system
### Description

Sometimes a maintainer needs to check the RP by running it on his local system. So we can add this new feature to demonstrate, how someone can review the PR by taking the code on his local system.

### Screenshots

_No response_

### PR DESCRIPTION
**Pull request by klaus0512**
## 🛠️ Fixes Issue

Closes #223  

## 👨‍💻 Changes proposed

Added a file to the "How to" section to demonstrate how one can Review a PR by taking the code on their system.

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/open-source-with-pradumna/pull/224"></a>

### CODE REVIEW

**Comment by klaus0512** [comment]
Hey! @Pradumnasaraf, I have linked the PR to the issue.
Kindly review it.

File: pages/How-to/guide/reviewing-pr-on-our-local-system.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+🔘 Now we have temporary track branch on our local system which we can check by `git branch` commmand. Now we can run the code on our local system.
+
+![11](https://user-images.githubusercontent.com/94779733/197357692-d1a29250-739f-406e-9fbf-38e6fa1dea4b.png)
+
+🔘 Now we can Reiview PR by opening it on our local system code editor.
+
+![12](https://user-images.githubusercontent.com/94779733/197358217-8993f611-7d02-47de-a1cc-689d7888086e.png)
+
+# Congratulation!! You have successfully reviewed a Pull Request on your local system. 🥳🚀
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/reviewing-pr-on-our-local-system.md:57]:
```suggestion
### Congratulation!! You have successfully reviewed a Pull Request on your local system. 🥳🚀
```

File: pages/How-to/guide/reviewing-pr-on-our-local-system.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# How to review a PR on our local system ?
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/reviewing-pr-on-our-local-system.md:1]:
```suggestion
## How to review a PR on our local system ?
```

File: pages/How-to/guide/reviewing-pr-on-our-local-system.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Review a PR on our local system allow our administrators or maintainers to check on the changes proposed in pull requests in there local system or request further changes before the pull request is merged by checking the working output on there local system.   
+    
+# Start Reviewing a PR on our local system.
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/reviewing-pr-on-our-local-system.md:5]:
```suggestion
## Start Reviewing a PR on our local system.
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the PR, Some changes required.

**Comment by klaus0512** [comment]
Hey, @Pradumnasaraf  
I have resolved the required changes, Kindly review it.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**